build/installers/liferay-workspace/liferay-workspace.xml

```diff
@@ -359,6 +359,17 @@
         <deleteFile>
             <path>${installdir}</path>
         </deleteFile>
+        <deleteFile>
+            <path>C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Liferay Workspace</path>
```

Comment:
I think this will be installed by default although it is empty. we don't put anything in it.
Default the start menu will be setup by windows os, very less cases user will change it.